Add example to 1.5 with empty result (#486)
ietf-wg-jsonpath/draft-ietf-jsonpath-base: draft-ietf-jsonpath-base.md

| `$.store.*`                               | all things in store, which are some books and a red bicycle  |
| `$.store..price`                          | the prices of everything in the store                        |
| `$..book[2]`                              | the third book                                               |
| `$..book[2].author`                       | the third book's author                                      |

> | `$..book[2].publisher`                    | empty result: there is no "publisher" member on third book   |

Reviewer: ```suggestion
| `$..book[2].publisher`                    | empty result: the third book does not have a "publisher" member |
```
